src/store/sagas/wallet.ts

```diff
@@ -144,22 +144,6 @@ export function* handleAirdrop(): Generator {
   }
 }
 
-export function* fetchSelectedTokensBalances(action: PayloadAction<string[]>): Generator {
-  const api = yield* getConnection()
-  const network = yield* select(networkType)
-  const walletAddress = yield* select(address)
-
-  const tokens = action.payload
-  const balances = yield* call(getTokenBalances, tokens, api, network, walletAddress)
-
-  const convertedBalances: ITokenBalance[] = balances.map(balance => ({
-    address: balance[0],
-    balance: balance[1]
-  }))
-
-  yield* put(actions.addTokenBalances(convertedBalances))
-}
-
 export function* fetchTokensBalances(): Generator {
```

Comment:
I think we still have some repeated logic here:
![image](https://github.com/invariant-labs/webapp-a0/assets/93533892/2484c133-9273-4f29-9f8f-15dcd2b902de)

what do you think about including `fetchTokensBalances` in `fetchBalances` saga.  the difference is that we also fetch native tzero tokens in `fetchBalances`. Btw I noticed that to improve can be also `init` saga. there is a duplicated setBalance() and fetchTokensBalances() after fetchBalances. you can check if that can be removed.